src/views/Home/components/Footer/index.view.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./footer.scss";
+
+const FooterComponent: React.FC = () => {
+  return (
+    <div className="Footer">
+      <div className="Footer__images">
+        <div>Team!</div>
+        <div>Links!</div>
+        <div>Scroll!</div>
+      </div>
+      <div className="Footer__text">
+        <div className="Footer__text--left">
+          Made with ❤️☕❤️ from Santa Cruz, California.
+        </div>
+        <div className="Footer__text--middle">
+          CruzHacks was formerly known as HackUCSC, and launched in April 2014.
+          The founders were&nbsp;
```

Comment:
SUPER NIT: I'm honestly not that bit a fan of having `&nbsp;` directly in the codebase as it creates a bit of messy code. We can instead [style the `a` tag accordingly ](https://techstacker.com/how-to-add-white-space-between-elements-react-jsx/). I'd prefer if we styled it using SCSS instead of hardcoding it into the text like that.

Reply:
I agree that these entities make code look a little unclean, but I'm not quite sure what about the resolution that you mentioned. The `&nbsp;` entities are here, because our linter requires that content within `a` tags be placed on a new line. Although I personally prefer inline `a` tags, this causes sentences to be partitioned weirdly and requires `&nbsp;` used in place of any trailing spaces that are pre-maturely truncated.

Let me know if this makes sense and how to proceed - also note that this section will be removed later when this sentence is integrated into our `Teams` or `Sponsors` page.

Reply:
Ahh yes! I see what the issue is. Since the `a` tag has 3 props and contents within it, in order to maintain the character count for that line to 120, it makes the `a` go to a separate line. Adding a space at the end of the line before the `a` tag would cause the linter to cry since we don't want trailing spaces. In order to remove these entities, we can [style it in the css](https://techstacker.com/how-to-add-white-space-between-elements-react-jsx/#or-just-use-css) by adding a `margin-left: .5rem` to the `a` tags.

Reply:
Ohh, that sounds like a clever fix! I'll have that done by tonight!